### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tschifftner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@ambimax/nimbus",
```
Comment (FaustTobias) [inline package.json:2]: ?

File: test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    const finalJson = jest.spyOn(fs, 'writeFileSync');
+    finalJson.mockImplementation();
+    await plugin.prepare(null, context);
+
+    expect(finalJson).toHaveBeenCalledWith(
+      expect.stringContaining('composer.json'),
+      expect.stringContaining('"version": "1.1.0"'),
+    );
+  });
+});
```
Comment (FaustTobias) [inline test/index.test.js:101]: Clean test structure! 👍

**Review by FaustTobias — CHANGES_REQUESTED** [review]

File: test/index.test.js (hunk 2 above)
Comment (tschifftner) [reply test/index.test.js:101]: Thanks

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        os:
+          - ubuntu-20.04
+    runs-on: '${{ matrix.os }}'
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Use Node.js ${{ matrix.node-version }}'
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ matrix.node-version }}'
+      - uses: bahmutov/npm-install@v1
```
[the remark this answers is not in the document]
Comment (FaustTobias) [reply .github/workflows/test.yml:29]: We decided on Slack that the action can be used instead of the official GitHub recommended method.

**Review by FaustTobias — APPROVED** [review]

**Pull request merged**